api-specificatie/openapi.yaml

```diff
@@ -81,23 +73,65 @@ paths:
         '500':
           $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/500"
-        '503':
-          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/503"
+        'default':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/default"
       tags:
       - volgindicatiepersonen
   /volgindicatiepersonen/{burgerservicenummer}:
+    get:
+      operationId: GetVolgindicatiePersoon
+      description: |
+        Opvragen specifiek abonnenement op een burgerservicenummer waarop een abonnee een volgindicatie heeft ingesteld.
+      summary: "raadpleeg specifieke volgindicatie"
+      parameters:
+        - in: path
+          name: burgerservicenummer
+          required: true
+          description: "Het burgerservicenummer, bedoeld in artikel 1.1 van de Wet\
+            \ algemene bepalingen burgerservicenummer. Alle nummers waarvoor geldt\
+            \ dat, indien aangeduid als (s0 s1 s2 s3 s4 s5 s6 s7 s8), het resultaat\
+            \ van (9*s0) + (8*s1) + (7*s2) +...+ (2*s7) - (1*s8) deelbaar is door\
+            \ elf. Er moeten dus 9 cijfers aanwezig zijn."
+          explode: false
+          schema:
+            $ref: "#/components/schemas/Burgerservicenummer"
+      responses:
+        200:
+          description: "OK"
+          headers:
+            api-version:
+              $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/headers/api_version"
+            warning:
+              $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/headers/warning"
+          content:
+            application/hal+json:
+              schema:
+                $ref: "#/components/schemas/VolgindicatiePersoon"
+        '400':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/400"
+        '401':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/401"
+        '403':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/403"
+        '404':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/404"
+        '406':
+          $ref: "https://raw.githubusercontent.com/VNG-Realisatie/Haal-Centraal-common/v1.0.0/api-specificatie/common.yaml#/components/responses/406"
+        '410':
```

Comment:
ik vraag me af of Gone relevant is hier en geleverd kan worden. De server maakt dan onderscheid tussen volgindicaties die nooit bestaan hebben (404) en volgindicaties die verwijderd zijn (410). Daarvoor moet ze ook in historie kijken. Ik betwijfel of dit onderscheid relevant is voor gebruikers.

Reply:
Eens! Wordt er sowieso wel historie bijgehouden op volgindicaties? Is het van belang om te weten op welk burgerservice een gebruiker ooit een volgindicatie heeft ingesteld? Ik vermoed van niet.